HACKING.md

```diff
@@ -16,11 +16,11 @@ various development dependencies.
 
 On Debian and recent Ubuntu ≥ 19.04:
 
-    $ sudo apt-get install nodejs npm
+    $ sudo apt-get install nodejs sassc
```

Comment:
Please keep npm. `nodejs` is a bit redundant as `npm` depends on it. But the `apt-get` and `dnf install` commands should have the same package names for consistency.

Reply:
Done while I was at it.